tests/jsm_test.ts

```diff
@@ -984,3 +984,27 @@ Deno.test("jsm - update consumer", async () => {
 
   await cleanup(ns, nc);
 });
+
+Deno.test("jsm - stream info subjects", async () => {
+  const { ns, nc } = await setup(jetstreamServerConf({}, true));
+  if (await notCompatible(ns, nc, "2.7.2")) {
+    return;
+  }
+  const jsm = await nc.jetstreamManager();
+  const name = nuid.next();
+  await jsm.streams.add({ name, subjects: [`${name}.>`] });
+
+  const js = nc.jetstream();
+  await js.publish(`${name}.a`);
+  await js.publish(`${name}.a.b`);
+  await js.publish(`${name}.a.b.c`);
+
+  const si = await jsm.streams.info(name, { subjects_filter: `>` });
```

Comment:
I would add a test after that where you ask for info without the subjects_filter field and assert that there is no list returned. Also, I may have use a more restrictive filter such as `${name}.a.>` and expect only 2 subjects in `si.state.subjects`?

Reply:
done.